topics/probot/index.md

```diff
@@ -0,0 +1,12 @@
+---
+created_by: Brandon Keepers, Bex Warner
+related: github-apps, bot, node-github
+display_name: Probot
+github_url: https://github.com/probot/
+logo: probot.png
+released: March 2017
+short_description: GitHub Apps to automate and improve your workflow.
```

Comment:
How about changing the short_description so it mentions Probot itself, not just the end result from using Probot? Maybe "Probot lets you create GitHub Apps to automate and improve your workflow."

Reply:
Done!